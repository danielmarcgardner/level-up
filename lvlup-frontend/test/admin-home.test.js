import AdminHome from '../src/components/Admin/admin-main-view/admin-main';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

test('AdminHome should render', () => {
  const adminhome = shallow(
    <AdminHome />,
  );
  expect(toJson(adminhome)).toMatchSnapshot();
});
