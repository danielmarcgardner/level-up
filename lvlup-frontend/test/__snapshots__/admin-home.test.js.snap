exports[`test AdminHome should render 1`] = `
<div
  className="lvl-table">
  <ChallengesHeader />
  <Connect(ChallengesTable) />
  <RequestsHeader />
  <Connect(RequestsTable) />
</div>
`;
